db/schema.rb

```diff
@@ -80,6 +80,9 @@
     t.integer "sport_facilities"
     t.float "working_time"
     t.float "students_per_teacher"
+    t.string "powiat", null: false
+    t.string "gmina", null: false
+    t.string "miejscowosc", null: false
```

Comment:
These names are specific to the polish law and I don't know an official translation for them. I don't think that there is a one to one translation for them that can be universally understood. For example a word "county" could be used to replace any of the above and it wouldn't be a translation mistake. Leaving them in polish allows us to see exctly what they mean. If any foreigners (highly unlikely) encounter them they would have to familiarise themselves at least somewhat with how the polish administrative division regardless of the name in the database. Can you think of a better naming convention?

Reply:
Yeah, good point. According to Wikipedia, "gmina" is most similar to "municipality" (https://en.wikipedia.org/wiki/Gmina), while "powiat" to "county" (https://en.wikipedia.org/wiki/Powiat), so maybe that's a way to go?
Another way of thinking about it could be that both "gmina" and "powiat" are both examples of specific levels of what's called administrative divisions (see list at https://en.wikipedia.org/wiki/List_of_administrative_divisions_by_country), so maybe we could have something like `administrative_division_level_1`, `administrative_division_level_2` etc.? That way, this part of our model would be country agnostic, which would be very cool, as that would make future app's globalization easier and can potentially attract contributors from other countries ;)

Reply:
We can start thinking about making it country agnostic when we specify which country we would like to expand to. Let's focus for now on making it in Poland.